op-deployer/pkg/deployer/standard/standard.go

```diff
@@ -189,9 +191,9 @@ func L1VersionsFor(chainID uint64) (L1Versions, error) {
 func GuardianAddressFor(chainID uint64) (common.Address, error) {
 	switch chainID {
 	case 1:
```

Comment:
we could collapse these cases into the same case

Reply:
what do you mean?